test_cacl_application.py::test_cacl_application_nondualtor missing iptable rules on supervisor

E       Failed: Missing expected iptables rules: set(['-A INPUT -i eth1-midplane -j ACCEPT'])

The test fails because it expects the iptable rule on the supervisor to be learned against `eth1-midplane` but it's learned against `br1`:
```
(Pdb) set(expected_iptables_rules) - set(actual_iptables_rules)
set(['-A INPUT -i eth1-midplane -j ACCEPT'])
(Pdb) set(actual_iptables_rules) - set(expected_iptables_rules)
set([u'-A INPUT -i br1 -j ACCEPT'])
```

This was introduced in https://github.com/sonic-net/sonic-buildimage/pull/16505 where `br1` had an alias of `eth1-midplane` added.
This causes the test to into the `if result` conditional and add these expected entries:
```
def append_midplane_traffic_rules(duthost, iptables_rules):
    result = duthost.shell('ip link show | grep -w "eth1-midplane"', module_ignore_errors=True)['stdout']
    if result:
        midplane_ip = duthost.shell('ip -4 -o addr show eth1-midplane | awk \'{print $4}\' | cut -d / -f1 | head -1',
                                    module_ignore_errors=True)['stdout']
        iptables_rules.append("-A INPUT -i eth1-midplane -j ACCEPT")
        iptables_rules.append("-A INPUT -s {}/32 -d {}/32 -j ACCEPT".format(midplane_ip, midplane_ip))
```

--- a/tests/cacl/test_cacl_application.py
+++ b/tests/cacl/test_cacl_application.py
@@ -364,7 +364,8 @@ def generate_and_append_block_ip2me_traffic_rules(duthost, iptables_rules, ip6ta
 
 def append_midplane_traffic_rules(duthost, iptables_rules):
     # Get the kernel intf name in the event that eth1-midplane is an altname
-    result = duthost.shell('ip link show eth1-midplane | awk \'NR==1{print$2}\' | cut -f1 -d"@" | cut -f1 -d":"', module_ignore_errors=True)['stdout']
+    result = duthost.shell('ip link show eth1-midplane | awk \'NR==1{print$2}\' | cut -f1 -d"@" | cut -f1 -d":"',
+                           module_ignore_errors=True)['stdout']
     if result:
         midplane_ip = duthost.shell('ip -4 -o addr show eth1-midplane | awk \'{print $4}\' | cut -d / -f1 | head -1',
                                     module_ignore_errors=True)['stdout']

--- a/tests/cacl/test_cacl_application.py
+++ b/tests/cacl/test_cacl_application.py
@@ -363,11 +363,12 @@ def generate_and_append_block_ip2me_traffic_rules(duthost, iptables_rules, ip6ta
 
 
 def append_midplane_traffic_rules(duthost, iptables_rules):
-    result = duthost.shell('ip link show | grep -w "eth1-midplane"', module_ignore_errors=True)['stdout']
+    # Get the kernel intf name in the event that eth1-midplane is an altname
+    result = duthost.shell('ip link show eth1-midplane | awk \'NR==1{print$2}\' | cut -f1 -d"@" | cut -f1 -d":"', module_ignore_errors=True)['stdout']
     if result:
         midplane_ip = duthost.shell('ip -4 -o addr show eth1-midplane | awk \'{print $4}\' | cut -d / -f1 | head -1',
                                     module_ignore_errors=True)['stdout']
-        iptables_rules.append("-A INPUT -i eth1-midplane -j ACCEPT")
+        iptables_rules.append("-A INPUT -i {} -j ACCEPT".format(result))
         iptables_rules.append("-A INPUT -s {}/32 -d {}/32 -j ACCEPT".format(midplane_ip, midplane_ip))
 
 

